### ISSUE TITLE
[BUG] - Kirby input does not fully center prefix

### ISSUE THREAD

**Issue by bdunkx** [kirbydesign/designsystem#3341]
- [x] I have written a [good issue](https://github.com/kirbydesign/designsystem/wiki/The-Good%3A-Issue) 

## Describe the bug
When using a kirby input along with a prefix, will the prefix not be fully centered as the input itself when using chrome. If Safari is used instead, then both prefix and input is centered.

## Describe how to reproduce the bug
1. Go to DRB in staging in safari, preferably jyskebank.
2. Enable both feature toggles: RE2: Enable international transfer & RE2: Enable international transfer for native
3. Go to the 'Transfer and pay' tab.
4. Click on the 'Transfer abroad'
5.a. Accept Terms and conditions if needed.
5.b. Select a Germany as country.
6. Click on 'Next' button.
7. Select 'Euro' as the currency.
8. Enter a amount and click 'Next'.
9. Enter some random numbers, approx. 10.

## Which Kirby version was used?
Kirby version: 9.0.0

## What was the expected behavior?
The prefix should have been centered as the input itself, and not slightly been more towards the bottom.

## Add any screenshots
![Screenshot 2024-01-18 at 12 26 53](https://github.com/kirbydesign/designsystem/assets/110385301/a65e0ff6-714c-4655-b0bf-4795b3529fab)

## Please complete the following information:
- OS: MacOS
- Browser Chrome
- Browser version 120.0.6099.234

OS: MacOS
Browser Safari
Browser version 17.2.1

## Are there any additional context?

<hr />

## Checklist:

The following tasks should be carried out in sequence in order to follow [the process of contributing](https://github.com/kirbydesign/designsystem/blob/main/.github/CONTRIBUTING.md/#the-process-of-contributing) correctly.

### Verification
To make sure the bug is not intended behaviour; it should be verified by a member of team Kirby before moving on to implementation. 

- [ ] Make sure the `NOT verified` label has been removed by a member of [team Kirby](https://github.com/kirbydesign/designsystem/blob/main/.github/SUPPORT.md#team-kirby); do not proceed until this is done

### Implementation 
The contributor who wants to implement this issue should: 

- [ ] Make sure you have read: "[Before you get coding](../CONTRIBUTING.md/#before-you-get-coding)".
- [ ] Signal to others you are working on the issue by assigning yourself.
- [ ] Create a branch from the [develop branch](https://github.com/kirbydesign/designsystem/tree/develop) following our [branch naming convention](https://github.com/kirbydesign/designsystem/wiki/The-Good%3A-Branch). 
- [ ] Create a test that reproduces the bug following guidelines in: "[The good: Test](https://github.com/kirbydesign/designsystem/wiki/The-Good%3A-Test)". 
- [ ] Write code to fix the bug

### Review
Once the issue has been implemented and is ready for review:

- [ ] Do a [self-review](https://github.com/kirbydesign/designsystem/wiki/The-Good%3A-Self-review). 
- [ ] Create a pull-request.

**Comment by RasmusKjeldgaard** [comment]
Closing this issue automatically because it has been open for more than 6 months and is not currently in progress. If this is still relevant, please feel free to reopen it or open a new issue with updated details. Thank you!
